deployments/openshift.yml

```diff
@@ -3,94 +3,99 @@
 # separate terms of service, privacy policy, and support
 # documentation.
 
-### The OpenShift Starter workflow will:
+# 💁 The OpenShift Starter workflow will:
 # - Checkout your repository
-# - Perform a Docker build
-# - Push the built image to an image registry
+# - Perform a container image build
+# - Push the built image to the GitHub Container Registry (GHCR)
 # - Log in to your OpenShift cluster
-# - Create an OpenShift app from the image and expose it to the internet.
-
-### Before you begin:
-# - Have write access to a container image registry such as quay.io or Dockerhub.
-# - Have access to an OpenShift cluster.
-#   - For instructions to get started with OpenShift see https://www.openshift.com/try
-# - The project you wish to add this workflow to should have a Dockerfile.
-#   - If you don't have a Dockerfile at the repository root, see the buildah-build step.
-#   - Builds from scratch are also available, but require more configuration.
-
-### To get the workflow running:
-# 1. Add this workflow to your repository.
-# 2. Edit the top-level 'env' section, which contains a list of environment variables that must be configured.
-# 3. Create the secrets referenced in the 'env' section under your repository Settings.
-# 4. Edit the 'branches' in the 'on' section to trigger the workflow on a push to your branch.
-# 5. Commit and push your changes.
-
-# For a more sophisticated example, see https://github.com/redhat-actions/spring-petclinic/blob/main/.github/workflows/petclinic-sample.yaml
-# Also see our GitHub organization, https://github.com/redhat-actions/
-# ▶️ See a video of how to set up this workflow at https://www.youtube.com/watch?v=6hgBO-1pKho
+# - Create an OpenShift app from the image and expose it to the internet
+
+# ℹ️ Before you begin:
+# 1. Have access to an OpenShift cluster. Refer to https://www.openshift.com/try
+# 2. The project you wish to add this workflow to should have a Dockerfile.
+#   - If you don't have a Dockerfile at the repository root,
+#     or if it is named differently (eg. 'Containerfile'), see the buildah-build step.
+#   - The buildah-build step can be replaced with any other image build step if you do not wish to use a Dockerfile.
+
+# ℹ️ This workflow requires some configuration.
```

Comment:
Reword this - the configuration is now all optional
the secrets just have to be created
https://cli.github.com/manual/gh_secret_set